AudioPlayer NPEs

Version 1.5.0.3
```
java.lang.NullPointerException: Attempt to read from field 'de.danoeh.antennapod.fragment.ChaptersFragment de.danoeh.antennapod.activity.AudioplayerActivity$AudioplayerPagerAdapter.chaptersFragment' on a null object reference
	at de.danoeh.antennapod.activity.AudioplayerActivity.onPositionObserverUpdate(AudioplayerActivity.java:2554)
	at de.danoeh.antennapod.activity.MediaplayerActivity$1.onPositionObserverUpdate(MediaplayerActivity.java:91)
	at de.danoeh.antennapod.core.util.playback.PlaybackController$MediaPositionObserver$1.run(PlaybackController.java:777)
	...
```

```
java.lang.RuntimeException: Unable to resume activity {de.danoeh.antennapod/de.danoeh.antennapod.activity.AudioplayerActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'java.util.List de.danoeh.antennapod.core.util.playback.Playable.getChapters()' on a null object reference
	...
Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'java.util.List de.danoeh.antennapod.core.util.playback.Playable.getChapters()' on a null object reference
	at de.danoeh.antennapod.activity.AudioplayerActivity$AudioplayerPagerAdapter.onMediaChanged(AudioplayerActivity.java:2071)
	at de.danoeh.antennapod.activity.AudioplayerActivity.onResume(AudioplayerActivity.java:171)
	at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1258)
	...
```

--- a/app/src/main/java/de/danoeh/antennapod/fragment/ChaptersFragment.java
+++ b/app/src/main/java/de/danoeh/antennapod/fragment/ChaptersFragment.java
@@ -68,7 +68,7 @@ public class ChaptersFragment extends ListFragment implements AudioplayerContent
         this.media = media;
         adapter.setMedia(media);
         adapter.notifyDataSetChanged();
-        if(media.getChapters() == null) {
+        if(media == null || media.getChapters() == null || media.getChapters().size() == 0) {
             setEmptyText(getString(R.string.no_items_label));
         } else {
             setEmptyText(null);

